README.md

```diff
@@ -65,8 +65,14 @@ val intArray = IntArray(3) // 0 0 0 - primitive int array
 
 ### Map
 ```Kotlin
-val map = mutableMapOf<String, Int>()
+val map = mutableMapOf<String, Int>()  //returns LinkedHashMap, maintains insertion order
```

Comment:
Thanks for your PR!

We try to keep things short here, so I have a few suggestions:
- hashMap["key"] = 0 //put operation - I believe, this is straightforward operation, so we can omit it here
- getOrDefault repeats line above
- Note: Set also works like HashMap -> Could you specify in what situation this is useful?

To summarize, thanks for your PR. I would propose to simplify it a bit considering my suggestions above:

```
val map = mutableMapOf<String, Int>()  // returns LinkedHashMap, maintains insertion order
val hashMap = HashMap<String, Int>() // returns HashMap, not maintains insertion order

val default = map.getOrDefault("key", 0)
```

Reply:
Thanks for your recommendations and changes.

> Note: Set also works like HashMap -> Could you specify in what situation this is useful?>

This will be helpful for below use case

- Track the visited items e.g. in graph
- Keep unique items in set & preserve the insertion order
- When just want to keep unique keys i.e. no need to store key->pair

Please let me know should I keep line `Note line of set` or not in my changes.

Reply:
Hi @gitam2869 !

I think we should omit the `Note line of set.`  The set is a well-known data structure. Potentially, we can have a separate set block in the cheat sheet if we find useful and unobvious API.

To summarize, I would propose to submit this code:

```
val map = mutableMapOf<String, Int>()  // returns LinkedHashMap, maintains insertion order
val hashMap = HashMap<String, Int>() // returns HashMap, not maintains insertion order

val default = map.getOrDefault("key", 0)
```

Reply:
Sure @IvanShafran , I agree with your suggestions. Making the changes as suggested by you. 

Please check the updated PR.